connector/dummy.go

```diff
@@ -82,8 +83,8 @@ func NewDummy(usernames []string, password string, period time.Duration, flags,
 	return conn
 }
 
-func (conn *Dummy) Authorize(username, password string) bool {
-	if password != conn.password {
+func (conn *Dummy) Authorize(username string, password []byte) bool {
+	if !reflect.DeepEqual(password, conn.password) {
```

Comment:
reflect is slow, for this you should use `bytes.Equal` (or `bytes.Compare` if you want to avoid converting back to strings)